src/db.cpp

```diff
@@ -32,7 +34,9 @@
 #include "similar.h"
 
 namespace bf = boost::filesystem;
+namespace bl = boost::locale;
```

Comment:
actually boost::filesystem is aliased here with bf as it provides a whole functionality family.
for the two methods translate() and format() it is more consistent if they would use the same mechanism. so, please use "using boost::locale::translate"